lib/classes/Variables.js

```diff
@@ -268,7 +268,7 @@ class Variables {
     return match.replace(
       this.variableSyntax,
       (context, contents) => contents.trim()
-    ).replace(/\s/g, '');
+    );
```

Comment:
I had to dive deep in the `git blame` rabbit hole.. but it seems you initially added the whitespace removal @eahefnawy. What are you thoughts on this PR?

Reply:
If I remember correctly, this is meant to normalize variables written like this `${opt: foo}`. I think the real solution would be to NOT remove white spaces from the literal fallback only, but keep it for variables. This might be easier said than done though...

Overall, imo the issue raised is more severe than cleaning the variable itself. So I wouldn't oppose losing that tiny functionality.